tests/services/test_harvest_dag_generator.py

```diff
@@ -23,7 +23,5 @@ def mock_get(key):
 def test_create_provider_dags(mock_variable):
     register_drivers()
     create_provider_dags()
-    assert list(harvest_dags().keys()) == list(fetch_catalog())
-    for provider in iter(list(fetch_catalog())):
-        dag = harvest_dags()[provider]
+    for dag in harvest_dags().values():
```

Comment:
Since there isn't a one-to-one match between DAGs and Providers in the catalog the list comparison no longer works. Similarly the provider `auc` no longer has an entry in the dictionary returned by `harvest_dags()` so the test was adjusted to look up the DAGs directly.